test/test_constructors.jl

```diff
@@ -62,6 +62,42 @@
                 @test Structure(fbr) == Structure(Tensor(Lvl{Int16}(Element(0.0))))
                 @test Structure(fbr) == Structure(Tensor(Lvl(Element(0.0), Int16(0))))
                 @test Structure(fbr) == Structure(Tensor(Lvl{Int16}(Element(0.0), 0)))
+
+                if key == "SingleList" || key == "SingleRLE"
+                    continue  # don't test similar for Single*
+                end
+                if key == "SparseBand"
```

Comment:
This one should be fixed shortly so we can remove this continue

Reply:
Done!